openbb_platform/core/openbb_core/app/static/package_builder.py

```diff
@@ -628,7 +628,8 @@ def format_params(
                     annotation=updated_type,
                     default=param.default,
                 )
-
+        if kwargs:
+            formatted["**kwargs"] = kwargs
         return MethodDefinition.reorder_params(params=formatted)
```

Comment:
To be consistent `reorder_params` should take care of the order

keeping the insertion
```
formatted["**" + name] = Parameter(
                    name="kwargs", kind=Parameter.VAR_KEYWORD, annotation=Any
                )
```

ensuring that `**kwargs` come last
```
    @staticmethod
    def reorder_params(params: Dict[str, Parameter]) -> "OrderedDict[str, Parameter]":
        """Reorder the params."""
        formatted_keys = list(params.keys())
        for k in ["provider", "extra_params", "**kwargs"]:
            if k in formatted_keys:
                formatted_keys.remove(k)
                formatted_keys.append(k)
        od: OrderedDict[str, Parameter] = OrderedDict()
        for k in formatted_keys:
            od[k] = params[k]
        return od
```